app/views/user/audits/index.html.erb

```diff
@@ -3,9 +3,20 @@
   <p>This shows when you have searched for and shared documents, and also when your documents have been viewed by people you shared them with.</p>
 
   <% if @audits.any? %>
-    <div class="index-cards">
-      <%= render @audits.sort_by(&:created_at).reverse! %>
-    </div>
+    <table class="table table-striped">
+      <thead>
+        <tr>
+          <th>Type</th>
+          <th>User</th>
+          <th>Time</th>
+          <th>Action</th>
+          <th>Comment</th>
+        </tr>
+      </thead>
+      <tbody>
+        <%= render @audits.sort_by(&:created_at).reverse! %>
```

Comment:
Is this an active record? Doing a big sort on the rails side can use a log of webserver memory when the database might already have this indexed and optimised. How about:
```@audies.order(created_at: :desc)``` instead